src/main/java/tuteez/logic/commands/RemarkCommand.java

```diff
@@ -16,10 +16,12 @@
 public abstract class RemarkCommand extends Command {
 
     public static final String COMMAND_WORD = "remark";
-    public static final String MESSAGE_USAGE = COMMAND_WORD
+    public static final String COMMAND_WORD_ALT = "rmk";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " (short form: " + COMMAND_WORD_ALT + ")"
```

Comment:
Could the example message be abit long especially with all the alt commands? Maybe we can consider shortening this or making it more concise. It is hard to view the entire message especially when the window is small.